src/main/java/io/javalin/Javalin.java

```diff
@@ -210,27 +307,55 @@ private void ensureActionIsPerformedBeforeServerStart(@NotNull String action) {
 
     // End embedded server methods
 
+    /**
+     * Defines an access manager for the instance. Secured endpoints require one to be set.
+     *
+     * @see <a href="https://javalin.io/documentation#access-manager">Access manager in docs</a>
+     */
     public Javalin accessManager(@NotNull AccessManager accessManager) {
         this.accessManager = accessManager;
         return this;
     }
 
+    /**
+     * Handles exceptions thrown in the handlers.
```

Comment:
Adds an exception-mapper to the instance. See https://javalin.io/documentation#exception-mapping for more details